src/filters/regex_manager.rs

```diff
@@ -87,14 +96,14 @@ impl RegexManager {
     pub fn cleanup(&mut self) {
         let now = self.now;
         for (_, v) in &mut self.map {
-            if now - v.last_used < std::time::Duration::from_secs(180) {
+            if now - v.last_used >= REGEX_MANAGER_DISCARD_TIME {
                 // Discard the regex to save memory.
                 v.regex = CompiledRegex::None;
             }
         }
     }
 
-    #[cfg(feature = "debug-info")]
+    #[cfg(any(feature = "debug-info", test))]
```

Comment:
we can leave these as `#[cfg(feature = "debug-info")]` - just make `mod tests` also `#[cfg(feature = "debug-info")]`.

Reply:
If we could suppose that `debug-info` is always enable for tests then it's okay.
I worried about the test coverage for configurations with !debug-info.

Reply:
Yes, CI runs with all features enabled

Reply:
Fixed.